engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details.controller.js

```diff
@@ -77,7 +77,7 @@
         });
 
         $scope.hideSyncButton = function(repository, advancedSync) {
-            var result = $scope.syncInProgress(repository.last_sync) || !repository.url || $scope.denied('sync_products', $scope.product);
+            var result = $scope.syncInProgress(repository.last_sync) || !repository.url || repository.mirroring_policy === 'mirror_complete' || $scope.denied('sync_products', $scope.product);
```

Comment:
Need to test it but this looks wrong maybe due to the name of the function hideSyncButton..Reading this code makes you think we hide sync button for mirror_complete repositories. Does it do that?

Reply:
If you just add your bitcoin wallet as the disabled reason for sync, it would level up from a bug to ransomware.. :stuck_out_tongue_closed_eyes:

Reply:
I think you may be right.. Let me look at that again..